src/main/java/com/rtb/projectmanagementtool/project/ProjectData.java

```diff
@@ -220,21 +212,16 @@ public boolean equals(Object other) {
     return other instanceof ProjectData && equals(this, (ProjectData) other);
   }
 
-  /** @return the string representation of this class in JSON format. */
+  /** @return the string representation of this class. */
   @Override
   public String toString() {
-    String json = "{";
-    json += "\"name\":";
-    json += "\"" + this.name + "\",";
-    json += "\"description\":";
-    json += "\"" + this.description + "\",";
-    json += "\"creatorId\":";
-    json += "\"" + this.creatorId + "\",";
-    json += "\"members\":";
-    json += "\"" + this.members.toString() + "\",";
-    json += "\"admins\":";
-    json += "\"" + this.admins.toString() + "\"";
-    json += "}";
-    return json;
+    String returnString = "{\n";
```

Comment:
nit:  using curly braces to wrap around stuff, it would be nice if the stuff inside were indented.